.hyperloop/checks/check-frontend-test-infrastructure.sh

```
@@ -21,11 +21,49 @@ set -euo pipefail
+  exit 0
+fi
+
 # If there are no UI source files, nothing to enforce.
+# NOTE: counts *.vue, *.ts, AND *.js — same file-type set as task_ui_changes above.
 ui_source_count=$(find "$UI_DIR" \
-  \( -name "*.vue" -o -name "*.ts" \) 2>/dev/null \
+  \( -name "*.vue" -o -name "*.ts" -o -name "*.js" \) 2>/dev/null \
   | grep -v node_modules \
-  | grep -v ".nuxt" \
+  | grep -v "\.nuxt" \
   | grep -v dist \
```

Shellcheck (0.11.0)

[style] Consider using 'grep -c' instead of 'grep|wc -l'.

(SC2126)